resources/views/event/display_includes/event_block.blade.php

```diff
@@ -1,4 +1,4 @@
-@if($event->mayViewEvent(Auth::user()))
+@if($event->mayViewEvent(Auth::user()) || Auth::user()->can('board'))
```

Comment:
Same here, you can add this to mayViewEvent

Reply:
Nope, mayViewEvent is now also used for the calendar and you do want to give the board the possibility to not have all events in their calendar.